Libraries/LibWeb/DOM/Node.cpp

```diff
@@ -132,10 +132,8 @@ void Node::dispatch_event(NonnullRefPtr<Event> event)
             dbg() << "calling event listener with this=" << this_value;
 #endif
             auto* event_wrapper = wrap(global_object, *event);
-            JS::MarkedValueList arguments(global_object.heap());
-            arguments.append(event_wrapper);
             auto& interpreter = document().interpreter();
-            (void)interpreter.call(function, this_value, move(arguments));
+            (void)interpreter.invoke(function, this_value, event_wrapper);
```

Comment:
This might introduce a behaviour change, I do not know if `global_object.heap()` is the same as `document().interpreter().heap()`, but if it's not...what are you even doing here?

Reply:
It's the same heap :)